src/app/modules/search-config/services/search-global-form-builder/search-global-form-builder.service.ts

```diff
@@ -62,6 +95,18 @@ export class SearchGlobalFormGroup extends ConfigFormGroup {
           10,
           1
         ),
+        searchConfigurations: new FormArray<SearchCollectionFormGroup>([], {
+          validators: Validators.required,
+        }),
+        customComponent: new ComponentFormControl(
+          SearchCollectionComponent,
+          {
+            searchConfigurations: () => this.customControls.searchConfigurations
+          },
+          {
```

Comment:
Is it normal to have this empty object ?

Reply:
Yes its a mandatory param in the function